language/move-compiler/src/inlining/translate.rs

```diff
@@ -285,6 +302,108 @@ impl<'l, 'r> Visitor for SubstitutionVisitor<'l, 'r> {
                     });
                     *id = SpecId::new(self.inliner.current_spec_block_counter);
                     self.inliner.current_spec_block_counter += 1;
+
+                    // after inlining, we should
+                    // - remove the function pointers in `used_locals`
```

Comment:
Can we pull the logic here in an independent function? (And perhaps section in this file?)

Reply:
Done

- The spec anchor rewriting logic is extracted to a separate function
- The lambda lifting logic is further extracted to a separate section in the same file